cypress_test/integration_tests/desktop/writer/track_changes_spec.js

```diff
@@ -42,7 +45,35 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Track Changes', function (
 
 	it('Accept All', function () {
 		helper.typeIntoDocument('Hello World');
+		for (var n = 0; n < 2; n++) {
+			cy.cGet('#tb_editbar_item_insertannotation').click();
+			cy.cGet('#annotation-modify-textarea-new').type('some text' + n);
+			cy.cGet('#annotation-save-new').click();
+			// Wait for animation
+			cy.wait(500);
+		}
```

Comment:
There is a function in desktop_helper called insertMultipleComment that can do this.

Reply:
Nah, that method uses a different UI (i.e: notebook bar) which also requires some additional UI setup, it was causing more problems and required existing tracked comment tests to change. This was the easy way around.